samples/bluetooth/iso_combined_bis_and_cis/prj.conf

```diff
@@ -0,0 +1,24 @@
+CONFIG_LOG=y
+CONFIG_LOG_BUFFER_SIZE=2048
+
+CONFIG_RING_BUFFER=y
+
+CONFIG_BT=y
+CONFIG_BT_ISO_BROADCASTER=y
+CONFIG_BT_ISO_CENTRAL=y
+
+CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT=1
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT=1
+CONFIG_BT_CTLR_CONN_ISO_STREAMS=1
+CONFIG_BT_DEVICE_NAME="Nordic_BIS_CIS"
```

Comment:
Should we align the name in the KConfig of this sample and the time_sync sample as they are being used, together?
I think that would improve the user experience and also makes testing easier.

Reply:
I will set CONFIG_BT_DEVICE_NAME="Nordic_ISO" for both samples